lang/cs/Org.Apache.REEF.All/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" />
+      </dependentAssembly>
+    </assemblyBinding>
+  </runtime>
+</configuration>
```

Comment:
You probably want to run check style to fix format. here new line required.

Apache RAT verifies that Apache headers are in place where needed. To run this check separately, execute:
mvn apache-rat:check
Checkstyle verifies that all Java code adheres to a coding standard. To run this check separately, execute:
mvn checkstyle:checkstyle